src/Faithlife.Utility/InvariantConvert.cs

```diff
@@ -113,6 +113,30 @@ public static string ToInvariantString(this double value)
 		/// <exception cref="FormatException">Failed to parse using invariant culture.</exception>
 		public static TimeSpan ParseTimeSpan(string text) => ThrowFormatExceptionIfNull(TryParseTimeSpan(text), text);
 
+		/// <summary>
+		/// Converts the value to a string using the invariant culture.
+		/// </summary>
+		public static string ToInvariantString(this object value)
+		{
+			switch (value)
+			{
+			case null:
+				return null;
+			case bool val:
+				return ToInvariantString(val);
```

Comment:
I'd rather these use the extension method if this method isn't an extension method to eliminate the smell of infinite recursion.

```suggestion
				return val.ToInvariantString();
```